apps/asn1parse.c

```diff
@@ -53,7 +53,7 @@ const OPTIONS asn1parse_options[] = {
     {"dump", OPT_DUMP, 0, "unknown data in hex form"},
     {"dlimit", OPT_DLIMIT, 'p',
      "dump the first arg bytes of unknown data in hex form"},
-    {NULL}
+    {NULL, 0, 0, NULL}
```

Comment:
I do not think we want this kind of changes.

Reply:
IMO this one is maybe worth having to be able to enable the error.